script/swap_and_bridge/test/README.md

```diff
@@ -0,0 +1,13 @@
+Tests in this folder are run as script as they require to be run in a Sepolia forked environment, since they depend on the contracts owned and deployed by Lido and Diva teams.
+
+Before running them, start an L1 and L2 fork node using the shell scripts `runL1TestNetwork.sh` and `runL2TestNetwork.sh` inside [script](../..) folder respectively and make sure to run the tests from this folder.
+
+Notice that these tests may fail fo reasons unrelated to the implementation of this feature, but rather because of problems with the target integration (for instance if the target LST contract does not mint new LSTs).
```

Comment:
Maybe mention somewhere in this README file that in case `L2_TOKEN_ADDR_DIVA` is not set inside `env` file, before each execution of `./e2e_test.sh` or `./integration_test.sh` both local networks need to be restarted otherwise deployment of Diva token will fail because it's already deployed.

Reply:
done